### PR TITLE
Update Quick-Start

### PR DESCRIPTION
**Pull request by YuriKravetc**

### CODE REVIEW

File: docs/Quick-Start.md [hunk 1]
```diff
@@ -2,7 +2,26 @@
...
+
+## Installation
+
+Single-server-VictoriaMetrics VictoriaMetrics is available as:
+
+* [Database as a Service (dbaas)](https://aws.amazon.com/marketplace/pp/prodview-4tbfq5icmbmyc)
+* [Docker images](https://hub.docker.com/r/victoriametrics/victoria-metrics/) 
+* [Snap packages](https://snapcraft.io/victoriametrics)
+* [Helm Charts](https://github.com/VictoriaMetrics/helm-charts#list-of-charts)
+* [Binary releases](https://github.com/VictoriaMetrics/VictoriaMetrics/releases),
```
Comment (hagen1778) [inline docs/Quick-Start.md:15]:
```suggestion
* [Binary releases](https://github.com/VictoriaMetrics/VictoriaMetrics/releases)
```

File: docs/Quick-Start.md [hunk 2]
```diff
@@ -2,7 +2,26 @@
...
 ---
 
-# Quick Start
+# Quick start
+
+## Installation
+
+Single-server-VictoriaMetrics VictoriaMetrics is available as:
+
+* [Database as a Service (dbaas)](https://aws.amazon.com/marketplace/pp/prodview-4tbfq5icmbmyc)
```
Comment (hagen1778) [inline docs/Quick-Start.md:11]:
```suggestion
* [Managed VictoriaMetrics at AWS](https://aws.amazon.com/marketplace/pp/prodview-4tbfq5icmbmyc)
```

File: docs/Quick-Start.md [hunk 3]
```diff
@@ -2,7 +2,26 @@
...
+* [Docker images](https://hub.docker.com/r/victoriametrics/victoria-metrics/) 
+* [Snap packages](https://snapcraft.io/victoriametrics)
+* [Helm Charts](https://github.com/VictoriaMetrics/helm-charts#list-of-charts)
+* [Binary releases](https://github.com/VictoriaMetrics/VictoriaMetrics/releases),
+* [Source code](https://github.com/VictoriaMetrics/VictoriaMetrics)
+* [How to build from sources](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-build-from-sources)
```
Comment (hagen1778) [inline docs/Quick-Start.md:17]:
```suggestion
* [Source code](https://github.com/VictoriaMetrics/VictoriaMetrics). See [How to build from sources](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-build-from-sources)
```

File: docs/Quick-Start.md [hunk 4]
```diff
@@ -2,7 +2,26 @@
...
+* [VictoriaMetrics on Linode](https://www.linode.com/marketplace/apps/victoriametrics/victoriametrics/)
+* [VictoriaMetrics on DigitalOcean](https://marketplace.digitalocean.com/apps/victoriametrics-single)
+
+Just download VictoriaMetrics and follow [these instructions](#how-to-start-victoriametrics).
+Then read [Prometheus setup](#prometheus-setup) and [Grafana setup](#grafana-setup) docs.
```
Comment (hagen1778) [inline docs/Quick-Start.md:22]: Can't find #prometheus-setup anchor here.

File: docs/Quick-Start.md [hunk 5]
```diff
@@ -2,7 +2,26 @@
...
+* [Docker images](https://hub.docker.com/r/victoriametrics/victoria-metrics/) 
+* [Snap packages](https://snapcraft.io/victoriametrics)
+* [Helm Charts](https://github.com/VictoriaMetrics/helm-charts#list-of-charts)
+* [Binary releases](https://github.com/VictoriaMetrics/VictoriaMetrics/releases),
+* [Source code](https://github.com/VictoriaMetrics/VictoriaMetrics)
+* [How to build from sources](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-build-from-sources)
+* [VictoriaMetrics on Linode](https://www.linode.com/marketplace/apps/victoriametrics/victoriametrics/)
+* [VictoriaMetrics on DigitalOcean](https://marketplace.digitalocean.com/apps/victoriametrics-single)
+
+Just download VictoriaMetrics and follow [these instructions](#how-to-start-victoriametrics).
```
Comment (hagen1778) [inline docs/Quick-Start.md:21]: Can't find #how-to-start-victoriametrics anchor here.

File: docs/Quick-Start.md [hunk 6]
```diff
@@ -2,7 +2,26 @@
...
+Then read [Prometheus setup](#prometheus-setup) and [Grafana setup](#grafana-setup) docs.
+
+### Starting VM-Signle with Docker:
```
Comment (hagen1778) [inline docs/Quick-Start.md:24]:
```suggestion
### Starting VM-Signle via Docker:
```

File: docs/Quick-Start.md [hunk 7]
```diff
@@ -2,7 +2,26 @@
 sort: 13
 ---
 
-# Quick Start
+# Quick start
```
Comment (hagen1778) [inline docs/Quick-Start.md:5]: Can we add details how to quick start with cluster version as well?

File: docs/Quick-Start.md [hunk 8]
```diff
@@ -13,9 +32,36 @@ docker run -it --rm -v `pwd`/victoria-metrics-data:/victoria-metrics-data -p 842
...
+* [Graphite-compatible agents such as StatsD](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-send-data-from-graphite-compatible-agents-such-as-statsd)
+* [OpenTSDB-compatible agents](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-send-data-from-opentsdb-compatible-agents)
+* [Prometheus remote_write API](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write)
+* [In JSON line format](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-data-in-json-line-format)
+* [In native format](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-data-in-native-format)
+* [Imported in CSV format](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-csv-data)
+* [Imported in Prometheus exposition format](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-data-in-prometheus-exposition-format)
+* `/api/v1/import` for importing data obtained from [/api/v1/export](#how-to-export-data-in-json-line-format).
```
Comment (hagen1778) [inline docs/Quick-Start.md:52]: Can't find `#how-to-export-data-in-json-line-format` anchor here.

**Review by hagen1778 — DISMISSED** [review]

File: docs/Quick-Start.md (hunk 7 above)
Comment (YuriKravetc) [reply docs/Quick-Start.md:5]: Will do, thanks

File: docs/Quick-Start.md (hunk 4 above)
Comment (YuriKravetc) [reply docs/Quick-Start.md]: Fixed Anchors, thanks!

File: docs/Quick-Start.md [hunk 9]
```diff
@@ -13,9 +31,46 @@ docker run -it --rm -v `pwd`/victoria-metrics-data:/victoria-metrics-data -p 842
...
+
+Data can be written by:
+
+* [DataDog agent](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-send-data-from-datadog-agent)
+* [InfluxDB-compatible agents such as Telegraf](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-send-data-from-influxdb-compatible-agents-such-as-telegraf)
+* [Graphite-compatible agents such as StatsD](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-send-data-from-graphite-compatible-agents-such-as-statsd)
+* [OpenTSDB-compatible agents](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-send-data-from-opentsdb-compatible-agents)
+* [Prometheus remote_write API](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write)
+* [In JSON line format](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-data-in-json-line-format)
+* [In native format](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-data-in-native-format)
```
Comment (valyala) [inline docs/Quick-Start.md:58]:
```suggestion
```

File: docs/Quick-Start.md [hunk 10]
```diff
@@ -13,9 +31,46 @@ docker run -it --rm -v `pwd`/victoria-metrics-data:/victoria-metrics-data -p 842
...
+
+```bash
+git clone https://github.com/VictoriaMetrics/VictoriaMetrics --branch cluster && cd VictoriaMetrics/deployment/docker && docker-compose up
+```
+
+* [Cluster setup](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#cluster-setup)
+
+## Writing data
+
+Data can be written by:
```
Comment (valyala) [inline docs/Quick-Start.md:50]:
```suggestion
Data can be written to VictoriaMetrics in the following ways:
```

File: docs/Quick-Start.md [hunk 11]
```diff
@@ -13,9 +31,46 @@ docker run -it --rm -v `pwd`/victoria-metrics-data:/victoria-metrics-data -p 842
...
+* [Prometheus remote_write API](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write)
+* [In JSON line format](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-data-in-json-line-format)
+* [In native format](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-data-in-native-format)
+* [Imported in CSV format](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-csv-data)
+* [Imported in Prometheus exposition format](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-data-in-prometheus-exposition-format)
+* `/api/v1/import` for importing data obtained from [/api/v1/export](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-export-data-in-json-line-format).
+ See [these docs](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-data-in-json-line-format) for details.
+
+## Reading data
+
```
Comment (valyala) [inline docs/Quick-Start.md:65]:
```suggestion

VictoriaMetrics various APIs for reading the data. [This document briefly describes these APIs](https://docs.victoriametrics.com/url-examples.html).

```

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
Thanks!

**Pull request merged**